app/static/src/app/components/options/SensitivityOptions.vue

```diff
@@ -5,18 +5,25 @@
         <button class="btn btn-primary mb-4 float-end" @click="toggleEdit(true)">Edit</button>
         <ul>
           <li><strong>Parameter:</strong> {{settings.parameterToVary}}</li>
-          <li><strong>Scale Type:</strong> {{ settings.scaleType }}</li>
+          <li v-if="settings.variationType !== 'Custom'"><strong>Scale Type:</strong> {{ settings.scaleType }}</li>
           <li><strong>Variation Type:</strong> {{ settings.variationType }}</li>
           <li v-if="settings.variationType === 'Percentage'">
             <strong>Variation (%):</strong> {{ settings.variationPercentage }}
           </li>
-          <template v-else>
+          <template v-if="settings.variationType === 'Range'">
             <li><strong>From:</strong> {{ settings.rangeFrom }}</li>
             <li><strong>To:</strong> {{ settings.rangeTo }}</li>
           </template>
-          <li><strong>Number of runs:</strong> {{ settings.numberOfRuns}}</li>
+          <li v-if="settings.variationType === 'Custom'">
+            <strong>Values:</strong>
```

Comment:
I was looking for some hint as to what my central value was. Do we want to encourage the users to bracket it?

Reply:
Yeah, I wondered about that. Do we want to enforce or just encourage?

Reply:
encourage seems enough - just knowing where it is would be nice